web/app/store/CommandTypes.js

```diff
@@ -19,17 +20,9 @@ Ext.define('Traccar.store.CommandTypes', {
     extend: 'Ext.data.Store',
     fields: ['type', 'name'],
 
-    listeners: {
-        'beforeload' : function (store) {
```

Comment:
We are setting it in `DevicesController` as extra param, ExtJS everything does for us, we do not need add this parameters manually.